### PR TITLE
trying to enable environmental parsing across the board in helmsman

### PR DESCRIPTION
**Pull request by tibers**
I am attempting to [fix this issue](https://github.com/Praqma/helmsman/issues/35). I am a total go novice and I need another set of eyes.

### CODE REVIEW

File: state.go [hunk 1]
```diff
@@ -159,8 +159,8 @@ func (s state) validate() (bool, string) {
 // if the env variable is empty or unset, an empty string is returned
 // if the string does not start with '$', it is returned as is.
 func subsituteEnv(name string) string {
```
Comment (davidarcher) [inline state.go]: Looks like this function essentially is the same thing as os.ExpandEnv so maybe just drop it completely?

Comment (tibers) [reply state.go]: I was going to make a "A WILD $THING APPEARS" joke but I couldn't find one SFW... I updated the PR because why not?

**Comment by tibers** [comment]
OK, fixed all the whitespace. Now there won't be some files with two spaces and some with five.

**Comment by jonbesga** [comment]
I assume this means that no dollar values can't be used then?

I have a password like `1111$2222` and it gets translated into the chart as 1111 because of this PR

**Comment by sami-alajrami** [comment]
@jonbesga you can escape the $ like so: `1111$$2222` or alternatively, if you are using v1.10.x (and don't need variable substitution anywhere else), you can use  the `--no-env-subst` flag to disable variable substitution everywhere.

**Pull request merged**